### DISCUSSION TITLE
The best way to use Polygon Edge is through Polygon Supernets. ?

### DISCUSSION THREAD

**Discussion by amandatsalis** [0xPolygon/polygon-edge#784] [General]
### Description

**What is the difference between Polygon Edgenet and Supernet?  Edgenet just a framework for creating blockchain and supernet is the same thing but premium version ?**

### Your environment

* OS and version : Ubuntu 18
* version of the Polygon Edge : v0.6.0
* branch that causes this issue : develop

hello, I am a student  who wants to learn more about the world of blockchain and web3. sorry i want to make sure i have created my own blockchain on localhost using polygon edge as documented, after i re-read there are polygon supernets which need validator at least _20.000 Matic_ https://polygon.technology/supernets/ form (https://docs. google.com/forms/d/11abPLDEEASA8qdG5MuaO9KxHsLapKywac2YNIoD8iyE/viewform?edit_requested=true) if I build my own ecosystem using polygon edge https://wiki.polygon.technology/docs/edge/get-started/set-up-ibft-on -the-cloud do I still pay as a validator 20,000 Matic? Or do I still create validators and invite people to join as validators? sorry if my question confuses you or i am wrong. Thank you!

**Comment by laviniat1996** [comment]
Hi @amandatsalis ,

Polygon Edge is a framework for creating your own blockchain where all validators are hosted by you. So, if all the validators belong to you, then your network is centralised and not as secure. This is where Supernetes comes in: it orchestrates professional validators to stake, and they get rewarded, in order to validate your network, thus making it more secure. To become a professional validator for Supernets, you have to stake a minimum of 20k Matic. 
If you wish to use Polygon Edge for production, please contact the Polygon team to discuss the details.
